src/components/game/game.js

```diff
@@ -0,0 +1,143 @@
+import Input from "./gameComponents/guessInput"
+import Hangman from "./gameComponents/hangman"
+import Display from "./gameComponents/letterDisplay"
+import Score from "./gameComponents/scoreRoundDisplay"
+import GuessDisplay from "./gameComponents/guessDisplay"
+import React  from "react"
+import {getRandomWord} from "../../utils"
+
+class Game extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      currWord: getRandomWord(),
+      guessedLetters: [],
+      displayLetters: [], 
+      numOfAllowedGuess: this.props.guess,
+      numOfGuess: 0,
+      userGuess: "",
+      currRound: 1,
+      rounds: this.props.rounds,
+      wins: 0,
+      isRoundDone: false,
+    };
+
+    this.updateRoundData = this.updateRoundData.bind(this);
+    this.nextGame = this.nextGame.bind(this)
+    this.restart = this.restart.bind(this)
+    this.isRoundComplete =this.isRoundComplete.bind(this)
+  }
+  
+  //Update Current Round Data
+  updateRoundData = (userGuess) => {
+    //Update Guess
+    let letters_guess = this.state.guessedLetters
+    let letters_display = this.state.displayLetters
+    let guessLeft = this.state.numOfGuess
+    let word = this.state.currWord
+    //reject repeated words
+     if (letters_guess.indexOf(userGuess) >= 0 && letters_guess.length !== 0) {
```

Comment:
```suggestion
     if (letters_guess && letters_guess.indexOf(userGuess) >= 0) {
```

everything above 0 is truthy